src/Domain/Model/MembershipApplication.php

```diff
@@ -38,9 +39,9 @@ public function __construct( ?int $id, string $type, Applicant $applicant, Payme
 		$this->applicant = $applicant;
 		$this->payment = $payment;
 		$this->donationReceipt = $donationReceipt;
-		$paymentMethod = $payment->getPaymentMethod();
-		if ( $paymentMethod instanceof BookablePayment ) {
-			$this->confirmed = $paymentMethod->paymentCompleted();
+		// TODO: Make isBooked() method public on BookablePayment
+		if ( $this->payment instanceof BookablePayment ) {
+				$this->confirmed = $this->payment->getValuationDate() !== null;
```

Comment:
We introduced the `confirmed` state when we introduced PayPal payments, because with external payments the membership application is unconfirmed until the user completes the payment.

With the PPL for memberships being dropped, we can mark this property as deprecated and set it to true by default.
When we reintroduce PPL payments, the export script in the FOC must check for payment state (booked/unbooked) anyway and not for membership application state.